modules/creating-a-drift-metric-using-cli.adoc

```diff
@@ -12,7 +12,9 @@ You can use the OpenShift command-line interface (CLI) to create a data drift me
 
 * Your OpenShift cluster administrator added you as a user to the {openshift-platform} cluster and has installed the TrustyAI service for the data science project that contains the deployed models.
 
+ifdef::upstream[]
 * You set up TrustyAI for your data science project, as described in link:{odhdocshome}/monitoring-data-science-models/#setting-up-trustyai-for-your-project_monitor[Setting up TrustyAI for your project].
+endif::[]
 
```

Comment:
Add another link for downstream:
ifndef::upstream[]
* You set up TrustyAI for your data science project, as described in link:{rhoaidocshome}{default-format-url}/monitoring_data_science_models/setting-up-trustyai-for-your-project_monitor[Setting up TrustyAI for your project].
endif::[]